src/js/helpers/language.js

```diff
@@ -0,0 +1,16 @@
+import Cookies from 'js-cookie';
+
+const getPathWithLangCode = (linkTo, lang) => {
```

Comment:
@ifsimicoded, I merged this without your review so I wanted to call this out for your attention. Instead of finding a solution at the routing level, I chose to make this helper function, that can take a path and an optional lang parameters.

It looks up the cookie lang settings is there is no lang passed and returns the full path with language code.